Formula/krb5.rb

```diff
@@ -25,12 +25,11 @@ class Krb5 < Formula
 
   uses_from_macos "bison"
 
-  on_linux do
-    depends_on "gettext"
-  end
-
   def install
     cd "src" do
+      # Prevent opportunistic linking to gettext.
+      ENV["ac_cv_search_dgettext"] = "no"
```

Comment:
`--disable-nls`?

Reply:
Ah I didn't see that.  Let me see if stops the gettext check.